onnxruntime/test/optimizer/graph_transform_utils_test.cc

```diff
@@ -48,9 +48,26 @@ TEST(GraphTransformerUtilsTests, TestGenerateGraphTransformers) {
 
   // Transformer name no match test. When there is no match empty list is expected.
   custom_list = {"EliminateIdentity"};
-  transformers = transformer_utils::GenerateTransformers(TransformerLevel::Level2, &custom_list);
+  transformers = transformer_utils::GenerateTransformers(TransformerLevel::Level2, custom_list);
   ASSERT_TRUE(transformers.size() == 0);
 }
 
+TEST(GraphTransformerUtilsTests, TestGenerateGraphTransformers_CustomList) {
```

Comment:
Can we make this a bit more complicated by having a case where we specify both a couple of rules and a transformer in one of the levels?
PS: It is nice that you check that there is no rule-based transformer added in level 2 in the current test.

Reply:
we don't have such a case right now... I will have to mock generate transformers to create such a case and it will defeat the point of the test. Once you start converting transformers to rules we can add this test...

Reply:
True, but we could add multiple rules. But no big deal, we check that in multiple other tests either way.